sycl/test/device-code-split/aot-accelerator.cpp

```diff
@@ -0,0 +1,4 @@
+// REQUIRES: aoc
+
+// RUN: %clangxx -fsycl -fsycl-targets=spir64_fpga-unknown-unknown-sycldevice -I %S/Inputs -o %t.out %S/split-per-source-main.cpp %S/Inputs/split-per-source-second-file.cpp
```

Comment:
fsycl-device-code-split=per_source option is needed here. Same for other aot tests.

Reply:
Thanks for the catch, applied